website/src/components/Footer/index.tsx

```diff
@@ -199,7 +199,7 @@ const Footer: React.FC = () => {
                             </div>
                             <div>
                                 <Link className={classes.contributeButton}>
-                                    <img loading="lazy" src="../images/logos/githubLogo.svg" className={classes.githubMobileIcon} alt={t('generic.github')}></img> 
+                                    <img loading="lazy" src="/images/logos/githubLogo.svg" className={classes.githubMobileIcon} alt={t('generic.github')}/>
```

Comment:
Is this tag proper ?

Reply:
Yes, it is. As explained in the PR description, we need to point `src` to the image folder at the root of the current sub directories so that we can call them in nested paths without having broken images

Reply:
I am concerned about the <img> tag here. Not the path

Reply:
Since `img` is an empty tag , we can close it immediately with ' />`